docs/tutorials/gs-ising.ipynb

```diff
@@ -1172,9 +1172,9 @@
     "        # the shape will now be (batches, 1, Nsites)\n",
     "        x = x.reshape(-1, 1, x.shape[-1])\n",
     "        \n",
-    "        x = nknn.DenseSymm(symmetries=graph.translation_group(),\n",
-    "                           features=self.alpha,\n",
-    "                           kernel_init=nn.initializers.normal(stddev=0.01))(x)\n",
+    "        x = nn.DenseSymm(symmetries=graph.translation_group(),\n",
+    "                         features=self.alpha,\n",
+    "                         kernel_init=nn.initializers.normal(stddev=0.01))(x)\n",
```

Comment:
I don't think `DenseSymm` should be changed to `flax`.

Reply:
ops